src/compiler/parse/read/style.ts

```diff
@@ -7,6 +7,7 @@ import { Style } from '../../interfaces';
 import parser_errors from '../errors';
 
 const regex_closing_style_tag = /<\/style\s*>/;
+const regex_starts_with_closing_style_tag = /<\/style\s*>/;
```

Comment:
maybe you want to do this?

```suggestion
const regex_starts_with_closing_style_tag = /^<\/style\s*>/;
```